src/Nethermind/Imapp.Benchmark.Runner/Program.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Linq;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Running;
+
+namespace Imapp.Benchmark.Runner
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var config = new ManualConfig()
+                .WithOptions(ConfigOptions.DisableOptimizationsValidator)
+                .WithOptions(ConfigOptions.DisableLogFile)
+                //.AddValidator(JitOptimizationsValidator.DontFailOnError)
+                .AddLogger(new BenchmarkNullLogger())
+                //.AddLogger(BenchmarkDotNet.Loggers.ConsoleLogger.Default)
+                .AddExporter(new BenchmarkNullExporter())
+                ;
+
+            var bytecode = "62FFFFFF600020";
```

Comment:
I prefer no default bytecode here. None of the other EVMs have this, let's keep the `bytecode` arg obligatory.

Reply:
Fixed